proto/options.proto

```diff
@@ -121,6 +121,7 @@ message CostingOptions {
   optional bool ignore_closures = 63;
   optional bool shortest = 64;
   optional float service_penalty = 65;
+  optional float use_distance = 66;
```

Comment:
new costing option from 0 to 1, how much do you care about minimizing over all distance